spec/requests/seances_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe 'Seances', type: :request do
+  let(:hall) { Hall.create(name: 'A', seats: 200) }
+  let(:movie) do
+    Movie.create(
+      title: 'Star Wars: Episode IV - A New Hope',
+      length: 150,
+      genre: 'Sci-Fi',
+      poster_url: 'https://upload.wikimedia.org/wikipedia/en/8/87/StarWarsMoviePoster1977.jpg'
+    )
+  end
+
+  before do
+    hall
+    movie
+  end
+
+  describe 'GET /seances' do
+    let(:seance) { Seance.create(date: '2021-06-01', time: '17:30', movie_id: movie.id, hall_id: hall.id) }
+
+    before { seance }
```

Comment:
I am not sure but for me it is similar to `let!(:seance)` which executes let immediately.
@Sonickers correct me if I am wrong.

Reply:
What it does is it runs the seance block once before every test below it

So that is why I can be sure that this seance is same for every test (same state, same attributes)